style.css

```diff
@@ -239,12 +286,18 @@ a {
 
 .code {
     overflow: auto;
+    display: flex;
+    flex-direction: column;
+    gap: 0.1rem;
+}
+
+.line {
     display: flex;
     flex-flow: row nowrap;
     align-items: center;
 }
 
-.code-box>.code>span {
+.code-box>.code>.line>span {
```

Comment:
avoid using very specific styling rather give a new class

Reply:
Thanks for the review bhai.